### PR TITLE
feat:cart page added

### PR DESCRIPTION
**Pull request by nyyshaaa**
https://deploy-preview-3--stupefied-wiles-6cf7dc.netlify.app/pages/cart/cart.html

### CODE REVIEW

**Review by AtulHande03a — APPROVED** [review]
looks good

File: pages/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,167 @@
+
+.cols2-grid{
+    display: grid;
+    width: 100%;
+    grid-gap: 2rem;
+    grid-template-columns:minmax(2rem,1fr) 2fr 1.5fr minmax(2rem,0.5fr);
+    grid-template-areas:
+        "... left-cart right-cart ...";
+    
+
```
Comment (pratik-kumar-sahu) [inline pages/cart/cart.css:10]: - get rid of uneven spacing

**Review by pratik-kumar-sahu — APPROVED** [review]
Else looks good go ahead with merging

File: pages/cart/cart.css (hunk 1 above)
Comment (nyyshaaa) [reply pages/cart/cart.css:10]: okeiy ,thanks!

**Pull request merged**